lib/matplotlib/ticker.py

```diff
@@ -800,14 +800,27 @@ class LogFormatter(Formatter):
     """
     Format values for log axis.
     """
-    def __init__(self, base=10.0, labelOnlyBase=True):
+    def __init__(self, base=10.0, labelOnlyBase=False,
+                 sublabel_filtering=False):
         """
         `base` is used to locate the decade tick, which will be the only
         one to be labeled if `labelOnlyBase` is ``True``.
+
+        Parameter
+        ---------
+        base : float, optional, default: 10.
```

Comment:
Got a slightly better error out of running it locally (only shows up on python2)

```
Exception occurred:
  File "/home/tcaswell/source/p/matplotlib/lib/matplotlib/contour.py", line 18, in <module>
    import matplotlib.ticker as ticker
  File "/home/tcaswell/source/p/matplotlib/lib/matplotlib/ticker.py", line 811
SyntaxError: Non-ASCII character '\xc2' in file /home/tcaswell/source/p/matplotlib/lib/matplotlib/ticker.py on line 812, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details

```

There is a non-breaking space in here:

```
             position: 25496 of 73929 (34%), column: 14
            character:   (displayed as  ) (codepoint 160, #o240, #xa0)
    preferred charset: unicode (Unicode (ISO10646))
code point in charset: 0xA0
               script: latin
               syntax: . 	which means: punctuation
             category: .:Base, b:Arabic, j:Japanese, l:Latin
             to input: type "C-x 8 RET a0" or "C-x 8 RET NO-BREAK SPACE"
          buffer code: #xC2 #xA0
            file code: #xC2 #xA0 (encoded by coding system utf-8-unix)
              display: by this font (glyph code)
    xft:-simp-Hack-normal-normal-normal-*-15-*-*-*-m-0-iso10646-1 (#x2A9)
       hardcoded face: nobreak-space

Character code properties: customize what to show
  name: NO-BREAK SPACE
  old-name: NON-BREAKING SPACE
  general-category: Zs (Separator, Space)
  decomposition: (noBreak 32) (noBreak ' ')

There are text properties here:
  face                 font-lock-doc-face
  fontified            t

```